v2/openapi/fb-provider-crosstransfer-api.yaml

```diff
@@ -0,0 +1,304 @@
+          type: string
+          example: '76a33ed9-8aa2-4781-93f8-d2369c0c2b14'
+        source:
+          $ref: '#/components/schemas/BridgingSourceParameters'
+        destination:
+          $ref: '#/components/schemas/BridgingDestination'
+    BridgingDestination:
+      type: object
+      required:
+        - chain
+        - currency
+        - address
+      properties:
+        chain:
+          $ref: '#/x-schemas/Blockchain'
+        currency:
+          $ref: '#/x-schemas/NativeCryptocurrency'
+        address:
+          description: Address of the destination account.
+          type: string
+        memo:
+          description: Memo field for the destination address. Required for some blockchains.
+          type: string
+    BridgingSourceParameters:
+      type: object
+      required: [ currency ]
+      properties:
+        chain:
+          $ref: '#/x-schemas/Blockchain'
+        currency:
+          $ref: '#/x-schemas/NativeCryptocurrency'
+        developerFee:
+          $ref: './fb-provider-api.yaml#/components/schemas/PositiveAmount'
+    CreateOnRampRoutingRequest:
+      type: object
+      required:
+        - idempotencyKey
+        - customerId
+        - source
+        - destination
+      properties:
+        idempotencyKey:
+          type: string
+          example: 76a33ed9-8aa2-4781-93f8-d2369c0c2b14
+        customerId:
+          type: string
+          example: 76a33ed9-8aa2-4781-93f8-d2369c0c2b14
+        source:
+          $ref: '#/components/schemas/OnRampSourceParameters'
+        destination:
+          $ref: '#/components/schemas/OnRampDestination'
+    OnRampDestination:
+      type: object
+      required:
+        - chain
+        - currency
+        - address
+      properties:
+        chain:
+          $ref: '#/x-schemas/Blockchain'
+        currency:
+          $ref: '#/x-schemas/NativeCryptocurrency'
+        address:
+          description: Address of the destination account.
+          type: string
+        memo:
+          description: Memo field for the destination address. Required for some blockchains.
+          type: string
+    OnRampSourceParameters:
+      type: object
+      required:
+        - currency
+      properties:
+        currency:
+          $ref: '#/x-schemas/NativeCryptocurrency'
+        developerFee:
+          $ref: '#/x-schemas/PositiveAmount'
+    OffRampSourceParameters:
+      type: object
+      required: [ chain, currency ]
+      properties:
+        chain:
+          $ref: '#/x-schemas/Blockchain'
+        currency:
+          $ref: '#/x-schemas/NativeCryptocurrency'
+        developerFee:
+          $ref: '#/x-schemas/PositiveAmount'
+        returnAddress:
+          type: string
+          description: >-
+            Blockchain address to return the funds in case of a failed transaction. 
+            MUST be on same blockchain network that is selected in the chain field.
+            Field will be ignored in case of Stellar chain.
```

Comment:
PublicBlockchainAddress object already defines a standard way to address blockchains. `returnAddress` should be PublicBlockchainAddress too.